#--------------------------------------------------------------------------------------------#
# Gazebo origin (lat-lon-alt) can be defined with 'sim_origin' param
#--------------------------------------------------------------------------------------------#
sim_origin: [37.558542, -5.931074, 7.89]

#--------------------------------------------------------------------------------------------#
# uav initial positions
#--------------------------------------------------------------------------------------------#

# uav_1_home
uav_1_home:
  home_frame_id: "uav_1_home"
  parent_frame: "map"
  units: "m"
  translation: [0.0, 0.0, 1.0]
  gz_initial_yaw: 0.0

# uav_2_home
uav_2_home:
  home_frame_id: "uav_2_home"
  parent_frame: "map"
  units: "m"
  translation: [-10.0, 10.0, 0.0]
  gz_initial_yaw: 0.0

# uav_3_home
uav_3_home:
  home_frame_id: "uav_3_home"
  parent_frame: "map"
  units: "m"
  translation: [10.0, 0.0, 0.0]
  gz_initial_yaw: 0.0